### ISSUE TITLE
[Suggestion] Allow Rolldoor controllers to be used in microcontrollers

### ISSUE THREAD

**Issue by XeonSquared** [PC-Logix/OpenSecurity#145]
While attempting to computerise the doors in my base, I discovered you can use the door controller in a microcontroller, removing the need for a relay to separate the computer from the network. When I tried this with a Rolldoor controller, it did not work.

Given that you can use a door controller in a microcontroller, I'd like to suggest allowing the roll door controller to be used in the same way, given the alternative - using it in a normal computer, with a relay attached to separate its component network from the network overall.

**Comment by unforseenconsequ3nces** [comment]
@ben-mkiv been wondering if you're ever going to implement this
(probably not cause it's been ages)

**Comment by ben-mkiv** [comment]
yea sorry, i haven't worked on minecraft mods for a few years. If anyone implements it and wants to make a PR they are free to do so

**Comment by unforseenconsequ3nces** [comment]
> yea sorry, i haven't worked on minecraft mods for a few years. If anyone implements it and wants to make a PR they are free to do so

yeah well, this is a really cool mod and I hope you'll continue working in it in the future for 1.12.2

**Comment by CaitlynMainer** [comment]
Added in f834d6fffba09648bc01259ca00640e77c3bf802
